hydromt/gis/raster.py

```diff
@@ -2286,9 +2286,9 @@ def interpolate_na(
         kwargs.update(dict(method=method, extrapolate=extrapolate))
         if dim0:
             interp_data = np.empty(self._obj.shape, dtype=self._obj.dtype)
-            for i, (_, sub_xds) in enumerate(self._obj.groupby(dim0)):
+            for i, (_, sub_xds) in enumerate(self._obj.groupby(dim0, squeeze=False)):
```

Comment:
Why did you change this?

Reply:
because it produces the following warning:

```
tests/gis/test_raster.py::test_interpolate_na
  /home/sam/Documents/work/hydromt/hydromt/gis/raster.py:2289: UserWarning: The `squeeze` kwarg to GroupBy is being removed.Pass .groupby(..., squeeze=False) to disable squeezing, which is the new default, and to silence this warning.
    for i, (_, sub_xds) in enumerate(self._obj.groupby(dim0)):
```